pages/getting-started/launching-the-game/clients-and-servers.mdx

```diff
@@ -0,0 +1,146 @@
+
+![runServer Gradle Task](/getting-started/clients-and-servers_1.png)
+
+<Subtitle>If the Gradle window is missing, you can open a new one in your Toolbar at the top <i>(View > Tool Windows > Gradle)</i></Subtitle>
+
+New files are generated in your project's `run` folder. However, an error should appear in your console.
+
+### Agree to EULA
+
+When launching the server for the first time, you need to agree to [Mojang's EULA](https://account.mojang.com/documents/minecraft_eula). A file called `eula.txt` has been created for that in the project's `run` folder. Change the EULA value to `true` in this file.
+
+```
+#By changing the setting below to TRUE you are indicating your agreement to our EULA (https://account.mojang.com/documents/minecraft_eula).
+#Sat Sep 03 12:03:44 CEST 2022
+eula=true
+```
+
+Run the `runServer` task for the second time. Now it also should be added to the `Select Run/Debug` dropdown element at the top right of your IDE, right next to the run button, for easier access.
+
+### Disable Server Online Mode
+
+In the development environment, the clients usually aren't logged in to any online accounts.
+
+![client 401 error status](/getting-started/clients-and-servers_2.png)
+
+<Subtitle>The client error of doom... or not...</Subtitle>
+
+Therefore, changes to the newly generated `server.properties` file in your project's `run` folder have to be made. The value for `online-mode` needs to be set to `false`. If you can't find it, use the `Search` tool to locate it <kbd>CTRL / CMD + F</kbd>.
+
+```
+# ...
+require-resource-pack=false
+use-native-transport=true
+max-players=20
+online-mode=false
+# ...
+```
+
+Now, the server won't try to authenticate the client accounts who are joining the server. The set-up of the server is finished. The `runServer` Gradle task is used to launch the server, and the usual server files are located in the project's `run` folder. You can interact with the server by using the server console.
+
+![server prompts](/getting-started/clients-and-servers_3.png)
+
+To join the server, use `localhost` as the `Server address` in-game.
+
+### Set up the Client profile
+
+Some testing requires multiple clients to interact in a multiplayer environment. Edit the `Minecraft Client` run configuration by pressing the white arrow. And modify the options to run multiple instances of this profile at the same time.
+
+![edit client configuration](/getting-started/clients-and-servers_4.png)
+
+![allow multiple instances](/getting-started/clients-and-servers_5.png)
+
+Now you can launch your client profile as many times as you want and connect multiple clients to your server. To check out all the instances and their consoles for logging and other purposes, switch the consoles with the tabs at the top of the console window.
+
+![consoles](/getting-started/clients-and-servers_6.png)
+</Steps>
+
+## Avoiding file issues
```

Comment:
This doesn't occur anymore as of Loom 1.1 and IntelliJ 2023

This section can just be deleted